bank-sdk/sdk/src/main/java/net/gini/android/bank/sdk/capture/digitalinvoice/view/DigitalInvoiceOnboardingIllustrationAdapter.kt

```diff
@@ -0,0 +1,43 @@
+package net.gini.android.bank.sdk.capture.digitalinvoice.view
+
+import android.view.View
+import android.view.ViewGroup
+import android.widget.ImageView
+import androidx.annotation.DrawableRes
+import androidx.annotation.StringRes
+import androidx.core.content.ContextCompat
+import net.gini.android.capture.view.InjectedViewAdapter
+
+interface DigitalInvoiceOnboardingIllustrationAdapter: InjectedViewAdapter {
+
+    fun onVisible()
+
+    fun onHidden()
+
+    fun onIllustrationSet(@DrawableRes drawableRes: Int)
+}
+
+class ImageDigitalInvoiceOnboardingIllustrationAdapter(): DigitalInvoiceOnboardingIllustrationAdapter {
```

Comment:
[Suggestion] Similarly to the above comment, the `ImageOnboardingIllustrationAdapter` is generic enough to be used in the Bank SDK as well.

If we make it public and reuse it, then we can reduce the amount of code.